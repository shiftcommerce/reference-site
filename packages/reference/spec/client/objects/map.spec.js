// Objects
import Map from '../../../client/objects/map'

test('renders correctly', () => {
  // act
  const wrapper = shallow(
    <Map/>
  )

  // assert
  expect(wrapper).toMatchSnapshot()
})
