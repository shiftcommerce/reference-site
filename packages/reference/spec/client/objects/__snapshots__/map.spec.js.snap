// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  className={Object {}}
  src="/static/googlemap.png"
/>
`;
